dali/ft/dalift.cmake

```diff
@@ -39,9 +39,14 @@ include_directories (
          ./../../system/mp 
          ./../base 
          ./../../system/include 
-         ./../../system/jlib 
-         ./../../common/environment
-         ./../../system/security/shared
+         ${HPCC_SOURCE_DIR}/system/jlib
+         ${HPCC_SOURCE_DIR}/rtl/include
+         ${HPCC_SOURCE_DIR}/rtl/eclrtl
+         ${HPCC_SOURCE_DIR}/common/deftype
+         ${HPCC_SOURCE_DIR}/ecl/hql
+         ${HPCC_SOURCE_DIR}/system/jhtree
+         ${HPCC_SOURCE_DIR}/common/environment
+         ${HPCC_SOURCE_DIR}/system/security/shared
```

Comment:
picky: I try to alpha sort these when making similar changes (makes it easier to avoid duplication which has happened in past)